package-lock.json

```diff
@@ -0,0 +1,6520 @@
+{
```

Comment:
Currently, we're using yarn for this repo, which is why there is no `package-lock.json`. Can you remove this for now?